Blog Post: fiatjaf Nostr LTS (#293)
OpenSats/website: data/blog/fiatjaf-recieves-lts-grant.mdx

---
title: "Long-Term Support For fiatjaf"
date: '2024-07-20'
tags: ['OpenSats', 'grants', 'nostr', 'LTS']
draft: false
authors: ['arvin', 'dergigi']
images: ['/static/images/blog/41-fiatjaf.jpg']
summary: "OpenSats Welcomes fiatjaf as an LTS Grantee."
---

OpenSats is proud to announce long-term support for fiatjaf, the innovative developer behind nostr.

Before nostr, fiatjaf's primary focus was on decentralized financial systems and tools for the Bitcoin Lightning Network, 
having contributed to projects like [lntxbot](https://github.com/fiatjaf/lntxbot), [c-lightning](https://github.com/ElementsProject/lightning/commits?author=fiatjaf), 
[lnbits](https://github.com/lnbits/lnbits/commits?author=fiatjaf), and creating [LNURL tools](https://github.com/fiatjaf/go-lnurl), to name a few.

When he is not writing inflammatory posts about how everybody is wrong, he spends his time tirelessly enhancing the [nostr](https://nostr.com/) ecosystem he pioneered.

> This LTS grant will enable fiatjaf to continue to improve the functionality and resilience of nostr through several ongoing projects, such as 

Reviewer: ```suggestion
This LTS grant will enable fiatjaf to focus all his time and attention on nostr and his many ongoing projects, such as 
```

[nostr-tools](https://github.com/nbd-wtf/nostr-tools), [nos2x](https://github.com/fiatjaf/nos2x), [go-nostr](https://github.com/nbd-wtf/go-nostr), 
[khatru](https://github.com/fiatjaf/khatru), [eventstore](https://github.com/fiatjaf/eventstore), [njump](https://njump.me/about), [loure](https://github.com/fiatjaf/loure), 
[wikistr](https://github.com/fiatjaf/wikistr), [patch34](https://github.com/fiatjaf/patch34), [nak](https://github.com/fiatjaf/nak), [gitstr](https://github.com/fiatjaf/gitstr), 

> [song](https://git.fiatjaf.com/song), [mmm](https://git.fiatjaf.com/mmm), and [promenade](https://git.fiatjaf.com/promenade). These tools and frameworks are essential for the 

Reviewer: ```suggestion
[song](https://git.fiatjaf.com/song), [mmm](https://git.fiatjaf.com/mmm), and [promenade](https://git.fiatjaf.com/promenade). These tools and frameworks are aimed at improving the 
```

development of nostr clients, building custom relays, and enhancing the overall nostr infrastructure​.

Additionally, fiatjaf will begin several new initiatives aimed at enhancing user experience, bolstering network resilience, and offering competitive alternatives to existing solutions. 
These projects include:

> -   NIP29 GTK4 desktop client

Reviewer: ```suggestion
-   NIP-29 GTK4 desktop client
```
